fili-navi/src/main/java/com/yahoo/bard/webservice/data/dimension/Tag.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.dimension;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * An interface for tag.
```

Comment:
Since this is tucked under the dimension package, and we only have 1 use for tags at the moment, I think it's OK to keep it tight. If we need to make it more widely used later, that should be easy enough to do.